tests/rptest/services/redpanda.py

```diff
@@ -992,6 +992,60 @@ def start_rp():
         self.start_service(node, start_rp)
         self._started.append(node)
 
+    def start_node_with_rpk(self, node, additional_args=""):
+        """
+        Start a single instance of redpanda using rpk. similar to start_node, 
+        this function will not return until redpanda appears to have started 
+        successfully.
+        """
+        node.account.mkdirs(RedpandaService.DATA_DIR)
+        node.account.mkdirs(os.path.dirname(RedpandaService.NODE_CONFIG_FILE))
+
+        def rpk_start():
+            # Pass environment variables via FOO=BAR shell expressions
+            env_preamble = " ".join(
+                [f"{k}={v}" for (k, v) in self._environment.items()])
+
+            cmd = (f"{env_preamble} {self.find_binary('rpk')} redpanda start"
+                   f" --config {RedpandaService.NODE_CONFIG_FILE} -v"
+                   f" {additional_args} "
+                   f" >> {RedpandaService.STDOUT_STDERR_CAPTURE} 2>&1 &")
+
+            node.account.ssh(cmd)
+
+        def is_status_ready():
+            status = None
+            try:
+                status = Admin.ready(node).get("status")
+            except requests.exceptions.ConnectionError:
+                self.logger.debug(
+                    f"node {node.name} not yet accepting connections")
+                return False
+            except:
+                self.logger.exception(
+                    f"error on getting status from {node.account.hostname}")
+                raise
+            if status != "ready":
+                self.logger.debug(
+                    f"status of {node.account.hostname} isn't ready: {status}")
+                return False
+            return True
+
+        def start_rp():
+            rpk_start()
+
+            wait_until(
+                is_status_ready,
+                timeout_sec=60,
+                backoff_sec=1,
+                err_msg=
+                f"Redpanda service {node.account.hostname} failed to start within {timeout} sec using rpk",
+                retry_on_exc=True)
+
+        self.logger.debug(f"Node status prior to redpanda startup:")
```

Comment:
no node status is printed here

Reply:
The next line will call `start_service`, which calls `_log_node_process_state` at the beginning.